### PR TITLE
updates to readme, screenshots, etc

### PR DESCRIPTION
**Pull request by emmakat**
after much trial and error, I've managed to sort out for myself how to use this awesome project. I have run through this a million times, and on Ubuntu 18.04, the /dev/sr0 _and_ dev/sg0 are required (for some reason). Without the /dev/sg0, it errors out and refuses to run. 
I added more documentation and screenshots to the readme. Feel free to use any or none of this - just wanted to share in case it helps the project. 
I do have the problem that discs aren't ejecting, so I'll try to solve that next. 
Thank you!

### CODE REVIEW

**Comment by rix1337** [comment]
@emmakat glad you found this project useful and thank you very much for the detailed Readme.

I need you to include the Screenshots in this PR and fix their links from your fork to this repo. After that I will merge. Thanks!

File: README.md [hunk 1]
```diff
@@ -12,22 +12,41 @@ Data-Disk | Uncompressed .ISO | ddrescue
...
-# Docker run
+```
+mkdir config rips
+```
+### (2) Find out the name(s) of the optical drive
+```
+lsscsi -g
+```
+For example, /dev/sr0 and /dev/sg0 are the two files that refer to a single optical drive. These names will be needed for the docker run command.  
+![lsscsi -g](https://github.com/emmakat/docker-ripper/blob/emmakat-patch-1/screenshots/lsscsi%20-g.png)
```
Comment (rix1337) [inline README.md:26]: wrong repo

File: README.md [hunk 2]
```diff
@@ -12,22 +12,41 @@ Data-Disk | Uncompressed .ISO | ddrescue
...
 ```
 docker run -d \
   --name="Ripper" \
   -v /path/to/config/:/config:rw \
   -v /path/to/rips/:/out:rw \
   --device=/dev/sr0:/dev/sr0 \
+  --device=/dev/sg0:/dev/sg0 \
   rix1337/docker-ripper
   ```
+  ![docker run](https://github.com/emmakat/docker-ripper/blob/emmakat-patch-1/screenshots/docker%20run.png)
```
Comment (rix1337) [inline README.md:39]: wrong repo

File: README.md [hunk 3]
```diff
@@ -12,22 +12,41 @@ Data-Disk | Uncompressed .ISO | ddrescue
...
+  
+  **To properly detect optical disk types in a docker environment this script relies on makemkvcon output.**
+
+MakeMKV is free while in Beta, but requires a valid license key. Ripper tries to fetch the latest free beta key on launch. Without a purchased license key Ripper may stop running at any time.
+
+If you have purchased a license key to MakeMKV/Ripper, after starting the container, go into the config directory you created, edit the file called `enter-your-key-then-rename-to.settings.conf`, and add your key between the quotes `app_Key = "`**[ENTER KEY HERE]**`"` then save and rename the file to settings.conf (then you can safely delete the original file called enter-your-key-then-rename-to.settings.conf)  
+![makemkv license](https://github.com/emmakat/docker-ripper/blob/emmakat-patch-1/screenshots/makemkvkey.png)
```
Comment (rix1337) [inline README.md:46]: wrong repo

File: README.md [hunk 4]
```diff
@@ -12,22 +12,41 @@ Data-Disk | Uncompressed .ISO | ddrescue
...
+At this point your config directory should look like this:  
+![config directory](https://github.com/emmakat/docker-ripper/blob/emmakat-patch-1/screenshots/config%20directory.png)
```
Comment (rix1337) [inline README.md:48]: wrong repo

**Review by rix1337 — CHANGES_REQUESTED** [review]
Please include the screenshots in this PR and fix their links to this repo

File: README.md (hunk 1 above)
Comment (emmakat) [reply README.md:26]: lsscsi -g

Comment (emmakat) [reply README.md:26]: currently trying to figure out how to do this. I read that I can make comments on each change, upload the image. But there's no clear instructions on How do I fix the PR that contains images inside a folder (the folder doesn't exist in the receiving repo). I think the markdown would reference https://github.com/rix1337/screenshots/lsscsi-g.png but I'm not sure. Perhaps I just need to do another branch that has the correct settings then make a new PR?

Comment (emmakat) [reply README.md:26]: or would the md be  https://github.com/rix1337/blob/screenshots/lsscsi-g.png

File: README.md (hunk 2 above)
Comment (emmakat) [inline README.md:39]: dockerrun

File: README.md (hunk 3 above)
Comment (emmakat) [inline README.md:46]: makemkvkey

File: README.md (hunk 4 above)
Comment (emmakat) [inline README.md:48]: configdirectory

**Comment by emmakat** [comment]
nevermind! I figured it out. Will send fix shortly

**Comment by emmakat** [comment]
closing this PR, and submitting a different one with the corrections. Thanks!

**Pull request closed without merging**